src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,14 @@
+export const namesHints = {
+  name: 'Unique identifier that may be configured to point to accounts, contracts, oracles, and NFTs.',
+  nameInAuction: 'Name that points to accounts, contracts, oracles, NFTs and more. Names shorter than 13 characters need to be auctioned.',
```

Comment:
I would use the same one as "name":
`Unique identifier that may be configured to point to accounts, contracts or oracles. Names shorter than` ...

Reply:
fixed